src/components/Questions/Body.vue

```diff
@@ -112,10 +117,12 @@ import {
   toRefs,
   computed,
   watch,
+  PropType,
   onMounted,
   onUpdated,
 } from "vue";
 import BaseIcon from "../UI/Icons/BaseIcon.vue";
+import { quizType } from "../../types";
```

Comment:
Why aren't we using absolute paths for files?

Reply:
it is because of how Typescript currently works with Vue.